models/anonymous_users.sql

```diff
@@ -1,23 +1,23 @@
 select
-  devices.device_id,
-  devices.device_user_agent as last_device_user_agent,
-  devices.device_type as last_device_type,
-  devices.device_operating_system as last_device_operating_system,
-  devices.device_browser as last_device_browser,
-  devices.device_browser_version as last_device_browser_version,
+  devices.id as device_id,
+  devices.user_agent as last_device_user_agent,
+  devices.type as last_device_type,
+  devices.operating_system as last_device_operating_system,
+  devices.browser as last_device_browser,
+  devices.browser_version as last_device_browser_version,
```

Comment:
I know we're still pre-1.0.0, but this would be a breaking change. Should we start tagging versions as we make updates to these models?

Reply:
Yes, I have been tagging minor releases for each of these PRs lately. DBT typically pins a version when installing dependencies so it shouldn't break existing users unless they have done something out of the ordinary.

I totally agree that post-1.0.0 all minor and patch releases should be backwards compatible.